taipy/gui/builder/_element.py

```diff
@@ -28,6 +29,7 @@ class _Element(ABC):
 
     _ELEMENT_NAME = ""
     _DEFAULT_PROPERTY = ""
+    __RE_INDEXED_PROPERTY = re.compile(r"^(.*?)_([\d]+)$")
```

Comment:
No need for the square brackets

Reply:
The square brackets for used for the declaration of the allowed chars, in this case numbers and is not included in the allowed chars